.github/workflows/rust-check-code-generation.yaml

```diff
@@ -96,7 +93,7 @@ jobs:
         uses: taiki-e/install-action@cargo-hack
       # intentionally no target specifier; see https://github.com/jonhoo/rust-ci-conf/pull/4
       - name: cargo hack
-        run: cargo hack --feature-powerset check
+        run: cargo hack --feature-powerset check -p astarte-message-hub-proto -p astarte-message-hub-proto-mock
```

Comment:
```suggestion
        run: cargo hack --each-features test --workspace
```

Reply:
This is faster than `--feature-powerset` while exercising all features and we include tests